swagger.json

```diff
@@ -1118,6 +1118,59 @@
             }
           }
         },
+        "/fleet/drivers/{driver-id}/log_edits": {
+            "get": {
+              "tags": [
+                "Fleet"
+              ],
+              "summary": "/fleet/drivers/{driver-id:[0-9]+}/log_edits",
+              "description": "Get a summary of the carrier edits for a specified driver. Note: This does not include edits made by driver",
+              "operationId": "getLogEdits",
+              "parameters": [
+                {
+                    "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                    "name": "driver-id",
```

Comment:
Could you make driver-id a parameter like accessTokenParam?  We use it so many places as a path param and a lot of the definitions are slightly inconsistent. ☹️   As an aside, it could probably be snake_case, since it's a path param and not a query param, but I think this is probably ok as spinal-case.

Reply:
changed back to snake_case